people-and-planet-ai/land-cover-classification/create_dataset.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This creates training and validation datasets for the model.
+
+As inputs it takes a Sentinel-2 image consisting of 13 bands.
+Each band contains data for a specific range of the electromagnetic spectrum.
+    https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S2_HARMONIZED
+
+As outputs it returns the probabilities of each classification for every pixel.
+The land cover labels for the training dataset come from the ESA WorldCover.
+    https://developers.google.com/earth-engine/datasets/catalog/ESA_WorldCover_v100
+"""
+
+from typing import Iterable, List, Optional, Tuple
```

Comment:
See PEP 585 comment below.

```suggestion
from collections.abc import Iterable
from typing import List, Optional
```